backend/spec/models/question_type_spec.rb

```diff
@@ -1,13 +1,37 @@
 require 'rails_helper'
 
 RSpec.describe QuestionType, type: :model do
-  describe 'uniqueness' do
-    subject { create(:question_type) }
+  subject { create(:question_type) }
 
+  describe 'uniqueness' do
     it { is_expected.to validate_uniqueness_of(:name).case_insensitive }
+  end
+
+  describe 'relationships' do
+    it { is_expected.to have_many(:questions).dependent(:destroy) }
+  end
+
+  describe '#destroy' do
+    describe 'When there are questions for question type' do
+      let!(:question_type) { create(:question).question_type }
+
+      it 'does not destroy' do
+        question_type.destroy
+        expect(question_type.errors.full_messages.first).to match("Can't destroy question type with 1 questions")
+      end
+
+      it { expect(described_class.count).to eq(1) }
```

Comment:
following the previous comment, this count should be zero

Reply:
why? when it can't destroy question_type it is supposed to be there right?

Reply:
actually, you are correct, it  should be 1 but you didn't called `destroy` before it

Reply:
done!